lib/js/src/protocol/SdlProtocolBase.js

```diff
@@ -470,17 +518,34 @@ class SdlProtocolBase {
     }
 
     /**
-     * Ends a session.
+     * Ends the default session.
      */
     endSession () {
-        const sessionID = this._getSessionId();
+        const sessionId = this._getSessionId();
         const hashID = this._hashID;
```

Comment:
Additional instances of improper variable naming patterns. Can address during a larger code cleanup prior to official release.